### DISCUSSION TITLE
On album und queue view the search field is automatically selected which pops up the tablet keyboard

### DISCUSSION THREAD

**Discussion by GK-774** [jcorporation/myMPD#751] [General]
On todays (May 5th) devel version when changing to the album view or the queue view, the cursor automatically is in the search field and this activates the keyboard on mobile devices (the keyboard pops up and hides half of the screen).
Is this intentional and/or can one deactivate it? It´s a bit unpractical when browsing through albums and on each page the keyboard pops up.

**Comment by jcorporation** [comment]
I opened an issue #752 for this.
